### ISSUE TITLE
Error on startup of 1.5

### ISSUE THREAD

**Issue by a9db0** [yakyak/yakyak#843]
Downloaded and unzipped the 1.5 build over my 1.44 build.  Started app with .\yakyak.  Returned the following error:

`~/Apps/yakyak-linux-x64 $ ./yakyak 
Gtk-Message: (for origin information, set GTK_DEBUG): failed to retrieve property `gtk-primary-button-warps-slider' of type `gboolean' from rc file value "((GString*) 0x37770064a280)" of type `gboolean'
A JavaScript error occurred in the main process
Uncaught Exception:
SyntaxError: Unexpected token  in JSON at position 0 at JSON.parse (<anonymous>)
    at loadFromFile (/home/a9db0/Apps/yakyak-linux-x64/resources/app/node_modules/tough-cookie-filestore/lib/filestore.js:148:32)
    at new FileCookieStore (/home/a9db0/Apps/yakyak-linux-x64/resources/app/node_modules/tough-cookie-filestore/lib/filestore.js:14:5)
    at Client.module.exports.Client.doInit (/home/a9db0/Apps/yakyak-linux-x64/resources/app/node_modules/hangupsjs/lib/client.js:188:68)
    at new Client (/home/a9db0/Apps/yakyak-linux-x64/resources/app/node_modules/hangupsjs/lib/client.js:102:12)
    at Object.<anonymous> (/home/a9db0/Apps/yakyak-linux-x64/resources/app/main.js:49:12)
    at Object.<anonymous> (/home/a9db0/Apps/yakyak-linux-x64/resources/app/main.js:591:4)
    at Object.<anonymous> (/home/a9db0/Apps/yakyak-linux-x64/resources/app/main.js:595:3)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)`

Wondering if you keep config info in a JSON file, and if so where that file is?  It could be that the file is corrupted.  Any pointer on where to look would be appreciated.

Dave
Linux Mint 18 / KDE

**Comment by averissimo** [comment]
Super strange, config directory is located at /home/<user>/.local/share/yakyak

Can you rename the directory and run again to see if it works?

This is starting to become a recurring error, we might have to fallback to delete the session and force a new one :\

**Comment by averissimo** [comment]
if renaming works can you please show the first few characters in cookies.json *(please don't show the full contents of the file as it can have personal information)*

**Comment by a9db0** [comment]
The bad news is there's no directory named yakyak in ~/.local/share/.  Which might explain the challenge of parsing a JSON file within. :)  I would think the code would create the dir / file if it didn't exist, which it apparently didn't do.  Anywhere else I should look?  If you can tell me the expected file name, I'll search my tree to see if it's wandered off somewhere else. (Stranger things have happened...)

**Comment by a9db0** [comment]
Oh, and thank you for the quick response!  I do appreciate it.

**Comment by averissimo** [comment]
yeah, that location was my mistake, it's on ~/.config/yakyak I edited it but seems like I was not fast enough :)

**Comment by a9db0** [comment]
Ah, there it is.  And it looks to be a very bad file.  Or, at least, it's not text JSON.  First few char are "^_��u8^_�b���q\��", without the quotes.   

That said, after I renamed the dir and restarted yakyak it fired up fine.  I had to log back in, and reset a few preferences, but that was a small price to pay.  Though I did get a humorous message to upgrade 1.5 to 1.4.3.

Thanks for your help.  If there's anything I can do to help troubleshoot, let me know.
